README.md

```diff
@@ -7,6 +7,15 @@ Useful resources:
 - [Notion wiki](https://www.notion.so/sfbrigade/Compass-source-of-truth-db118a6d51344dccadc49d1988662d66)
 - [Slack channel](https://sfbrigade.slack.com/archives/C01R8E75N1K)
 
+## Filing a new issue
+
+If you run into an issue or would like to request a feature, please file an issue.
```

Comment:
For an issue that requests a new feature, it may require both UX design work and engineering work. What label(s) are appropriate in that situation?

Reply:
it sounds like we may no longer need these labels. if so, we could get rid of them and the prefixes (at least moving forward)

@tam-janice, given the new board structure, will we no longer have separate eng and dev tickets? asking b/c traditionally the tickets were either prefixed with [ENG] or [UX] (or alternatively assigned similar labels).

Reply:
we could also repurpose the labels for "engineering"-only (tech stuff, chores) and "UX"-only tickets (design that isn't meant to be handed off to dev), which would distinguish them from a story/task that goes through the whole lifecycle.

Reply:
Here's what I'm thinking:

- We remove the prefixes [eng] and [UX]
- We will keep the column 'in design' 
- If something needs design, it'll jsut be assigned to me and I'll take care of it
- there will never be a ticket that reqires both UX and eng. If it's an eng ticket design should already be done

Does this work?